### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by darahayes**
### Description

* Logout button wasn't working because `isOnline` wasn't being passed down. It was always showing the `you are offline error`. It's not a big deal but do we even need to check that? You should be able to log out while offline IMO.
* Logged in user's name was being displayed as two first names

##### Checklist

- [ ] `npm test` passes
- [ ] `npm run build` works
- [ ] tests are included
- [ ] documentation is changed or added

### CODE REVIEW

**Comment by wtrocki** [comment]
When offline logout will fail with dinosaur which is bad experience so @kingsleyzissou added a nice message there. 

It will be cool to pass `isOnline` but if that is a lot of work then-current solution is ok (we will need to remove dialog as well)

**Comment by darahayes** [comment]
Okay good to know. Current solution is perfect then we were just forgetting to pass `isOnline` in there. The actual functionality works perfectly!

**Comment by darahayes** [comment]
update: I've added a fix here that enables keycloak with self signed certs

**Comment by wtrocki** [comment]
Looks good from my side

File: server/.env [hunk 1]
```diff
@@ -10,4 +10,8 @@ MQTT_HOST=
 MQTT_PORT=
 MQTT_PASSWORD=
 MQTT_USERNAME=
-MQTT_PROTOCOL= 
\ No newline at end of file
+MQTT_PROTOCOL= 
+
+# Hack to enable keycloak with self signed certs
+# don't do it in production
+NODE_TLS_REJECT_UNAUTHORIZED=0
```
Comment (wtrocki) [inline server/.env:17]: Typically this can be pushed without people knowing it. We had that in readme before

**Pull request merged**